tools/test-build

```diff
@@ -62,6 +62,7 @@ sphinx_html()
 {
     (
         cd docs/sphinx
+        export SPHINXOPTS="-W"
```

Comment:
I copied over the other Sphinx target defining SPHINXOPTS but you are right this is clearer. Will push a commit.